ArmEmulator/MemTests.fs

```diff
@@ -921,6 +975,26 @@ module MemTests
             let registers = [R0, 2u; R1, 0x100u] |> Map.ofList
             let flags = { N= false; C=false; Z=false; V=false}
             {Fl= flags; Regs= registers; MM = memory}
+        let baseDataPath2 =
```

Comment:
These datapaths are almost identical, can you not make one a default record and then edit it?